packages/contract-artifacts/src/lint_properties.ts

```diff
@@ -0,0 +1,61 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import * as artifacts from './index';
+
+export const RequiredProperties: string[] = [
+    'compilerOutput.evm.bytecode.object',
+    'compilerOutput.evm.deployedBytecode.object',
```

Comment:
@xianny I thought you said that `deployedBytecode` was not ultimately necessary for EVM execution? Did this change?

Reply:
Yup, we need `deployedBytecode` for EVM execution.